server/market/market.go

```diff
@@ -100,6 +103,8 @@ type Market struct {
 	swapper Swapper
 	auth    AuthManager
 
+	// checkUnspentBase  func(coinID []byte) (val uint64, confs int64, err error)
+	// checkUnspentQuote func(coinID []byte) (val uint64, confs int64, err error)
```

Comment:
Can remove, but this was a though I had before deciding on `Backend.VerifyUnspentCoin`